05_run_ocp.sh

```diff
@@ -117,6 +117,6 @@ ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman run \
     -d --net host --privileged --name ironic -v /run:/run:shared -v /dev:/dev localhost/ironic
 
 # Create a master_nodes.json file
-cat ~stack/ironic_nodes.json | jq '.nodes[0:3] |  {nodes: .}' | tee ocp/master_nodes.json
+cat $WORKING_DIR/ironic_nodes.json | jq '.nodes[0:3] |  {nodes: .}' | tee ocp/master_nodes.json
```

Comment:
this will slightly break people with existing setups, but I think we can live with that at this stage

Reply:
true, didn't consider that with all the buildup/teardown :/

Reply:
```suggestion
jq '.nodes[0:3] |  {nodes: .}' "${WORKING_DIR}/ironic_nodes.json" | tee ocp/master_nodes.json
```